### ISSUE TITLE
ollama run llama2 >>> /show license: No such file or directory

### ISSUE THREAD

**Issue by Keredu** [jmorganca/ollama#776]
I started using ollama and tried to run llama2. It works fine but when I tried to use the command `/show license`, I got an "No such file or directory" error.:
```
ollama run llama2
>>> /show license
error: couldn't get model
Error: stat /home/ubuntu/.ollama/models/manifests/registry.ollama.ai/library/llama2/latest: no such file or directory
```

It it's running on a new ubuntu EC2 machine (without GPU) and ollama was installed using:
```
curl https://ollama.ai/install.sh | sh
```

**Comment by Keredu** [comment]
@BruceMacD this is fixed in #778 , should I close this issue?

**Comment by BruceMacD** [comment]
Thanks for the update
